addons/account/models/account_account.py

```diff
@@ -38,7 +38,7 @@ def _check_account_type_unique_current_year_earning(self):
     name = fields.Char(string="Account Name", required=True, index='trigram', tracking=True, translate=True)
     currency_id = fields.Many2one('res.currency', string='Account Currency', tracking=True,
         help="Forces all journal items in this account to have a specific currency (i.e. bank journals). If no currency is set, entries can use any currency.")
-    code = fields.Char(size=64, required=True, tracking=True)
+    code = fields.Char(size=64, required=True, tracking=True, index=True)
```

Comment:
Not sure if the prefix searches use the index here @xmo-odoo . 
If you have unaccent enabled (by default on SaaS/OdooSH?), and you also use the `ilike` operator to do your prefix search: `('code', '=ilike', '1111%')`  will be translated to `"code" like unaccent('1111%')`, and it won't match your btree index (without unaccent). Right ?

Two solutions:
- Add `unaccent=False` to the `code` field (not sure if unaccent can help the user to find his account, it is only numbers, right?).
- Change expression.py to not use unaccent on '=like/=ilike' operators anymore. IMO, the `unaccent` feature was designed to help people to find records without considering accents in names/titles into account. But I am pretty sure that we cannot use/select `=like/=ilike` operators in the webclient (anymore?). Then most of the usage of `=like/=ilike` is technical and makes no sense to use unaccent. This idea is probably crazy and there are probably counter-examples, what do you think?

Reply:
> not sure if unaccent can help the user to find his account, it is only numbers, right?

In the vast majority of cases, yes. But not always: sometimes there are letters and/or dashes or dots.

> Change expression.py to not use unaccent on '=like/=ilike' operators anymore.

What? Why wouldn't it be used anymore? It is used, especially for `code` because we only search codes by prefix.